### PR TITLE
fix incompatible api

### PR DESCRIPTION
**Pull request by CrazyJvm**
simple.go cannot run properly because of incompatible api.

### CODE REVIEW

File: _examples/simple.go [hunk 1]
```diff
@@ -14,11 +14,11 @@ func main() {
 	abortOn(err)
 	defer client.Close()
 
-	err = produce(client, 10000)
+	err = produce(&client, 10000)
```
Comment (dim) [inline _examples/simple.go]: No, this isn't right, please remove the `&` here

File: _examples/simple.go [hunk 2]
```diff
@@ -14,11 +14,11 @@ func main() {
...
 	abortOn(err)
 	log.Println("Produced 10000 messages.")
 
-	err = consume(client, 1000, func(message *sarama.ConsumerMessage) {
+	err = consume(&client, 1000, func(message *sarama.ConsumerMessage) {
```
Comment (dim) [inline _examples/simple.go]: No, this isn't right, please remove the `&` here

File: _examples/simple.go [hunk 3]
```diff
@@ -29,7 +29,7 @@ func main() {
 }
 
 func produce(client *sarama.Client, n int) error {
```
Comment (dim) [inline _examples/simple.go]: `sarama.Client` is now an interface in the latest sarama version, please change to `func produce(client sarama.Client, n int) error`

File: _examples/simple.go [hunk 4]
```diff
@@ -29,7 +29,7 @@ func main() {
...
-	producer, err := sarama.NewSyncProducerFromClient(client)
+	producer, err := sarama.NewSyncProducerFromClient(*client)
```
Comment (dim) [inline _examples/simple.go]: No need for `*`

File: _examples/simple.go [hunk 5]
```diff
@@ -47,7 +52,7 @@ func produce(client *sarama.Client, n int) error {
 func consume(client *sarama.Client, n int, cb func(*sarama.ConsumerMessage)) error {
 	// Connect consumer
 	config := cluster.Config{CommitEvery: time.Second}
-	consumer, err := cluster.NewConsumerFromClient(client, []string{"localhost:2181"}, "my-group", "my-topic", &config)
+	consumer, err := cluster.NewConsumerFromClient(*client, []string{"localhost:2181"}, "my-group", []string{"my-topic"}, &config)
```
Comment (dim) [inline _examples/simple.go]: No need for `*`

**Comment by CrazyJvm** [comment]
thanks, modify according to the latest samara api.

**Pull request merged**